romtools/__init__.py

```diff
@@ -72,9 +72,9 @@
   - ROM/FOM exploitation via sampling
   - ROM/FOM exploitation via Dakota-driven sampling
 
-# Demos/HOWTOs
+# Demos/tutorials
 
-In this section, we present some demos/examples showing how to use `romtools` in practice.
+Please see https://pressio.github.io/rom-tools-and-workflows-demos for demos and tutorials
 
 TBD
```

Comment:
Can we remove the TBD now?

Reply:
Yes please! If you see any other egregious things please feel free to change!

Reply:
Ok I removed it